### PR TITLE
notebooks: surface alert in compute block for incomplete results

### PR DESCRIPTION
**Pull request by rvantonder**
Pops an alert so you know when results are incomplete if the query didn't finish in 1 minute.

Screen Shot 2022-03-18 at 2 08 08 PM

## Test plan
Manually tested for experimental feature.

### CODE REVIEW

**Comment by camdencheek** [comment]
how many compute credits do I get?

**Comment by rvantonder** [comment]
at least 12

File: enterprise/cmd/frontend/internal/compute/streaming/stream.go [hunk 1]
```diff
@@ -115,6 +115,13 @@ LOOP:
 		_ = eventWriter.Event("error", streamhttp.EventError{Message: err.Error()})
 		return
 	}
+
+	if err := ctx.Err(); err == context.DeadlineExceeded {
```
Comment (camdencheek) [inline enterprise/cmd/frontend/internal/compute/streaming/stream.go:119]: very nit:
```suggestion
	if err := ctx.Err(); errors.Is(err, context.DeadlineExceeded) {
```

it really doesn't make a difference here, but I'd rather avoid having any examples in the codebase of comparing errors with `==`

File: client/web/src/notebooks/blocks/compute/component/src/Main.elm [hunk 2]
```diff
@@ -766,6 +815,11 @@ darkModeTextInputColor =
     E.rgb255 0x1D 0x22 0x2F
 
 
+alertBackgroundColor : E.Color
+alertBackgroundColor =
+    E.rgb255 0x9C 0x65 0x00
```
Comment (rvantonder) [inline client/web/src/notebooks/blocks/compute/component/src/Main.elm:820]: I can CSS-ify this sort of thing with our named variables. Still low stakes change so will do that if/when it's needed. I did take this color from the Wild Card design system for warnings color.

**Review by camdencheek — APPROVED** [review]

File: enterprise/cmd/frontend/internal/compute/streaming/stream.go (hunk 1 above)
Comment (rvantonder) [reply enterprise/cmd/frontend/internal/compute/streaming/stream.go:119]: nice, I was looking for this code and couldn't remember it!

**Pull request merged**